src/GlobalStyles.js

```diff
@@ -0,0 +1,17 @@
+import { createGlobalStyle } from 'styled-components'
+
+const GlobalStyles = createGlobalStyle`
+* {
+        box-sizing: border-box;
+		margin: 0;
+		padding: 0;
+		font-family: "Montserrat", Arial, sans-serif;
+		color: inherit;  
+}
+
+body {
+		height: 100%;
```

Comment:
I would suggest to set the height to `100vh`. That way it's a bit more accurate than `100%`.